apps/prepopulate/app_prepopulate.py

```diff
@@ -79,7 +80,10 @@ def prepopulate_data(file_name, default_user=get_default_user()):
                 if not res:
                     raise Exception()
             else:
-                ids = service.post([data])
+                try:
+                    ids = service.post([data])
+                except werkzeug.exceptions.Conflict:
+                    ids = [data.get('_id')]  # data with given id is there already
```

Comment:
refactored this, there were items without id with conflicts, so handling this case now too